config/samples/cost-mgmt_v1alpha1_costmanagement.yaml

```diff
@@ -3,5 +3,3 @@ kind: CostManagement
 metadata:
   name: costmanagement-sample
 spec:
```

Comment:
whenever I deploy this weith the empty spec, I get this error: 
```
$ oc apply -f config/samples/cost-mgmt_v1alpha1_costmanagement.yaml
The CostManagement "costmanagement-sample" is invalid: spec: Invalid value: "null": spec in body must be of type object: "null"
```
this is resolved by removing the spec field

Reply:
removed the `spec:` line.